Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -410,12 +421,17 @@ final class DeclarationTests: XCTestCase {
 
     assertParse(
       """
-      private(1️⃣get, set2️⃣, didSet) var a = 0
+      private(1️⃣get, set, didSet) var a = 0
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code 'get,' in modifier"),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "unexpected code ', didSet' in modifier"),
-      ]
+        DiagnosticSpec(
+          message: "expected 'set' in modifier",
+          fixIts: ["replace 'get, , didSet' with 'set'"]
```

Comment:
This fix-it message looks a bit strange for me.

Reply:
I agree. Maybe this can be fixed at the same time as my comment above that the Fix-It should say “remove” instead of “replace”. So this could become `remove 'get,' and ', didSet'`

Reply:
I used the `RemoveNodesFixIt` but it doesn't include the "and". It format it as a comma separated list. 
We could make a new one or update the existing one. I think the latter would be the best.

Reply:
I wanted to know what was going wrong here and you nerd-sniped me into this. Fixing it in https://github.com/apple/swift-syntax/pull/1651. With that PR it will say `remove 'get,' and ', didSet'`.